isis/src/base/objs/Cube/Cube.cpp

```diff
@@ -133,11 +133,39 @@ namespace Isis {
     std::ifstream isdStream(isdFile.expanded().toStdString());
     std::ifstream labelStream(labelFile.expanded().toStdString());
 
-    Pvl label;
+    if (isdStream.fail()) {
+      QString msg = QString("failed to open isd stream: %1").arg(isdFile.expanded());
+      throw IException(IException::Io, msg,
+                 isdFile.baseName().toStdString().c_str(), 153);
+    }
```

Comment:
this is good because I would sometimes mess up the path to the ISD and I get a bogus error from the JSON lib. Would this also trigger in the case of a file not existing? (presumably yes).

Reply:
Yeah, it also triggers if the file doesn't exist, which was the issue I was running into!